lib/http/request.js

```diff
@@ -307,11 +307,8 @@ class HttpRequest extends EventEmitter {
   }
 
   setProxyIfNeeded() {
-    if (this.httpOpts.proxy !== null) {
-      // TODO: remove proxy-agent as dependency
-      const ProxyAgent = require('proxy-agent');
-      let proxyUri = this.httpOpts.proxy;
-      this.reqOptions.agent = new ProxyAgent(proxyUri);
+    if (this.params.desiredCapabilities && this.params.desiredCapabilities.proxy!== null) {
```

Comment:
missing some spaces there.

Reply:
How does the `this.params.desiredCapabilities.proxy` look here? Maybe we should keep the old way as well.